### ISSUE TITLE
Improve accessibility of CodeMirror in Custom HTML widget and theme/plugin editors

### ISSUE THREAD

**Issue by westonruter** [WordPress/better-code-editing#37]
Once keyboard focus is inside of CodeMirror, it is trapped and cannot be escaped. A fix for this has been implemented for CodeMirror in the Customizer's Additional CSS (see #34) by allowing a user to hit `ESC` followed by `Tab` or `Shift`+`Tab` to move forward or behind. The same thing needs to be implemented for CodeMirror.

Alternatively, maybe there is already a way in CodeMirror to escape from an editor with just the keyboard, but I could not find a way.

Additionally, should there be help text added to the Custom HTML widget and plugin/theme editors as was added for Additional CSS in #34?

**Comment by westonruter** [comment]
The `Esc` plus `Tab` is actually already advised on the Theme and Plugin editors:

![image](https://user-images.githubusercontent.com/134745/29700439-54d8b802-8919-11e7-9b51-d3cfb68cff8c.png)

So this is the right approach.
